modules/kn-service-apply.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * serverless/knative_serving/serverless-applications.adoc
+// * serverless/cli_reference/kn-services-ref.adoc
+
+[id="kn-service-apply_{context}"]
+= Applying service declarations
+
+You can declaratively configure a Knative service by using the `kn service apply` command. If the service does not exist it will be created, otherwise the existing service will be updated with the options that have been changed.
+
+The `kn service apply` command is especially useful when used in an automated way, such as in a shell script or a continuous integration pipeline, because in these types of applications users typically want to fully specify the state of the service in a single command to declare the target state.
```

Comment:
Something about _when used in an automated way_ bugs me a bit. Not sure how to fix it, or if you need to. 
Should this be two sentences, the second starting with _In these types_. I can see why you have it as one. It just seems like a long sentence.

Reply:
I wonder, too, if you should swap the sentences around, to define what `kn service update` does, then show how it can be _especially useful__?
Something like:
_You can use the `kn service update` command to make specific changes to a service. In contrast to the kn service apply command, when using the kn service update command you only have to specify the changes that you want to update, rather than the full configuration for the Knative service. (I love this sentence, really points out the difference!) The `kn service apply` command is especially useful..._

Reply:
Made some updates, lmk what you think 🙂